build.gradle

```diff
@@ -135,6 +135,7 @@ ext {
   composeActivity = "androidx.activity:activity-compose:1.6.1"
 
   accompanistFlowLayout = "com.google.accompanist:accompanist-flowlayout:0.28.0"
+  accompanistPager = "com.google.accompanist:accompanist-pager:0.28.0"
```

Comment:
can you add an `accompanistVersion` property and use it for both accompanist libraries to ensure they stay in sync?

Reply:
added👍🏻